src/AppBundle/Action/DefaultAction.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace AppBundle\Action;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class DefaultAction
+{
+    use ControllerTrait;
+
+    private $container;
```

Comment:
@vincentchalamon :+1: , PHP doc above a property allows do not scan the whole file to find what type is it. The question here is not that IDE can do it for you but improve code readability for other developers. BTW, I suppose not everyone IDE can guess it.